libs/config_engine.py

```diff
@@ -185,6 +186,9 @@ def get_area_all(self):
         area_all = next(area for area in areas if area.id == ALL_AREAS)
         return area_all
 
+    def get_area_config_path(self, area_id):
```

Comment:
Wonder why this appears as changed here. This is already on master

Reply:
IDK maybe because I resolved conficlts?